admin-ui/app/dialogs/send-push-notification.html

```diff
@@ -14,7 +14,16 @@ <h4 class="modal-title" id="myModalLabel">Send push to Application One</h4>
       </div>
     </div>
 
-
+      
```

Comment:
With Android you can manually put the device into deep sleep mode for testing, so for me it is useful :)

Also it was in the jiras (https://issues.jboss.org/browse/AGPUSH-1577) (the relationship between the author of the PR and the author of the JIRA not withstanding)